src/persist-client/src/batch.rs

```diff
@@ -430,13 +430,13 @@ pub(crate) const BLOB_TARGET_SIZE: Config<usize> = Config::new(
 
 pub(crate) const INLINE_WRITES_SINGLE_MAX_BYTES: Config<usize> = Config::new(
     "persist_inline_writes_single_max_bytes",
-    0,
+    4096,
     "The (exclusive) maximum size of a write that persist will inline in metadata.",
 );
 
 pub(crate) const INLINE_WRITES_TOTAL_MAX_BYTES: Config<usize> = Config::new(
     "persist_inline_writes_total_max_bytes",
-    0,
+    1 * MiB,
```

Comment:
@bkirwi / @ParkMyCar can you weigh in on these?

Reply:
`persist_inline_writes_single_max_bytes` and `persist_inline_writes_total_max_bytes` both LGTM